helmclient.go

```diff
@@ -85,17 +101,35 @@ func (c *Client) DeleteRelease(releaseName string, options ...helmclient.DeleteO
 // values provided when the chart was installed. The releaseName is the name
 // of the Helm Release that is set when the Helm Chart is installed.
 func (c *Client) GetReleaseContent(releaseName string) (*ReleaseContent, error) {
-	t, err := c.newTunnel()
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
-	defer c.closeTunnel(t)
+	var err error
 
-	resp, err := c.newHelmClientFromTunnel(t).ReleaseContent(releaseName)
-	if IsReleaseNotFound(err) {
-		return nil, microerror.Maskf(releaseNotFoundError, releaseName)
-	} else if err != nil {
-		return nil, microerror.Mask(err)
+	var resp *hapiservices.GetReleaseContentResponse
+	{
+		o := func() error {
+			t, err := c.newTunnel()
+			if err != nil {
+				return microerror.Mask(err)
+			}
+			defer c.closeTunnel(t)
+
+			resp, err = c.newHelmClientFromTunnel(t).ReleaseContent(releaseName)
+			if IsReleaseNotFound(err) {
+				return backoff.Permanent(releaseNotFoundError)
```

Comment:
I think that additional context information of `releaseName` should be added to wrapped error here. It was present in earlier version. So maybe this should be:
```go
return backoff.Permanent(microerror.Maskf(releaseNotFoundError, releaseName))
```

Reply:
I thought about this but wasn't sure. I can do this though.

Reply:
Done.